source/MRMesh/MRMesh.vcxproj.filters

```diff
@@ -1209,9 +1209,6 @@
     <ClInclude Include="MRMeshAttributesToUpdate.h">
       <Filter>Source Files\MeshAlgorithm</Filter>
     </ClInclude>
-    <ClInclude Include="MRBindingMacros.h">
-      <Filter>Source Files\Basic</Filter>
-    </ClInclude>
```

Comment:
why?

Reply:
Moved to MRPch because I need it in `MRPch/MRFmt.h`.